mcp/src/bugbug_mcp/server.py

```diff
@@ -55,7 +56,8 @@ async def patch_review(
     else:
         raise ValueError(f"Unsupported patch URL: {patch_url}")
 
-    return get_code_review_tool().generate_initial_prompt(patch, "TEXT")
+    # FIXME: add the system prompt as well
```

Comment:
File a follow-up issue so we don't forget

Reply:
Filed https://github.com/mozilla/bugbug/issues/5564